src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const rows = Array.from(table.rows);
+
+rows.forEach(row => {
+  if (row.cells.length > 1) {
+    const secondCell = row.cells[1];
+    if (secondCell) {
+      const clonedCell = secondCell.cloneNode(true);
+      if (row.cells.length > 2) {
+        const lastCell = row.cells[row.cells.length - 1];
+        if (lastCell) {
+          row.insertBefore(clonedCell, lastCell);
+        } else {
+          row.appendChild(clonedCell);
+        }
+      } else {
+        row.appendChild(clonedCell);
+      }
+    }
+  }
```

Comment:
The logic of this block of code is a bit confusing. It seems like you are trying to clone the second cell of each row and insert it before the last cell or append it as the last cell if there are less than 3 cells. But there is some unnecessary code here. For example, the check `if (secondCell)` is not needed because if a row has more than one cell, the second cell will always exist. Similarly, the check `if (lastCell)` is also not needed because if a row has more than two cells, the last cell will always exist. You can simplify this block of code by removing these unnecessary checks.